addon/styles/post.css

```diff
@@ -0,0 +1,45 @@
+.post-content-header h1 {
```

Comment:
can't we use the css helpers for font size here anyway? 🤔 https://ember-styleguide.netlify.app/concepts/typography